src/CLI/klish/patches/klish-2.1.4/clish/shell/shell_tinyrl.c.diff

```diff
@@ -0,0 +1,327 @@
+14a15
+> #include <unistd.h>
+19a21,23
+> #include "clish/plugin/clish_api.h"
+> //#include "clish/plugin/mgmt_clish_alias.h"
```

Comment:
Let's just delete the code rather than commenting out.

Reply:
Done